BankingApp/src/com/company/entities/Conturi.java

```diff
@@ -0,0 +1,120 @@
+package com.company.entities;
+
+import java.util.Scanner;
+
+//o clasa mai de sine statatoare pentru main, cu metode in ea, urmeaza de impartit in entities si service
+
+public class Conturi {
```

Comment:
de preferat sa eviti sa folosesti romana in numele claselor si campurilor, good practice e sa se foloseasca engleza, de multe ori vei lucra cu echipe internationale, sau in proiecte pt alte tari iar dupa ce vei face proiectul il vei preda altora, care trebuie sa inteleaga ce scrie acolo

Reply:
numele clasei sa nu fie la plural niciodata